### PR TITLE
WSGNN

### PR DESCRIPTION
**Pull request by dyxg**

### CODE REVIEW

File: opengsl/method/gnnsolver.py
Comment (zzysh12345) [inline opengsl/method/gnnsolver.py:1]: put it in gslsolver

File: opengsl/method/models/wsgnn.py [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+            x = lin(x)
+            x = F.relu(x, inplace=True)
+            if self.use_bn:
+                x = self.bns[i](x)
+            x = F.dropout(x, p=self.dropout, training=self.training)
+        x = self.lins[-1](x)
+        return x
+
+
+class DenseAPPNP(nn.Module):
```
Comment (zzysh12345) [inline opengsl/method/models/wsgnn.py:67]: Can you replace this part with our implemented APPNP?

File: opengsl/method/gnnsolver.py
Comment (dyxg) [reply opengsl/method/gnnsolver.py:1]: fixed

File: opengsl/method/models/wsgnn.py (hunk 1 above)
Comment (dyxg) [reply opengsl/method/models/wsgnn.py]: It is replaced now

**Pull request merged**